src/samples/article-history/v1/preprint-only.json

```diff
@@ -0,0 +1,10 @@
+{
+    "versions": [
+        {
```

Comment:
Currently, each of the versions of an article can be accessed at the endpoint `/articles/[article-id]/version/[version-number]`. Introducing versions without version numbers seems wrong.

I suggest that we add another optional top level property of `preprint` with the required properties of `uri` and `date` and optional `description` property.

Reply:
> Currently, each of the versions of an article can be accessed at the endpoint /articles/[article-id]/version/[version-number]. Introducing versions without version numbers seems wrong.

It's not a known version, you won't be able to access it at `/articles/[article-id]/version/[version-number]`

> I suggest that we add another optional top level property of preprint with the required properties of uri and date and optional description property.

Do you mean like:

```json
{
    "preprint": "2015-12-15",
    "preprint-description": "...",
    "received": "2016-01-02",
    "accepted": "2016-02-28",
    "versions": [
          ...
    ]
}
```

It's possible for an article to have multiple preprints so that won't work. [Discussion ticket here](https://github.com/elifesciences/issues/issues/4702). 

This change is modelling an 'event' in the article's history. There are other event types planned, for now we're capturing the publication a preprint.

Reply:
I was thinking of a single extra property of `preprint` which was an object but it could also be an array. It was really because it felt uncomfortable to introduce "versions" that would not be accessible at the `/articles/[article-id]/versions/[version-id]` endpoint.

In practice will we be updating our XML each time that a preprint version has been deposited on bioRxiv? Or will we only be referring to the latest preprint?

Reply:
it wouldn't be a pre-print then but a post-print ...

Reply:
> it wouldn't be a pre-print then but a post-print

Fair point.

Another question then. In practice will we be listing all versions of preprint that have been deposited or only the latest one?

Reply:
It will be possible for an article to have multiple preprints attached to it, I assume from different pre-print servers. I've not read anything about just a single pre-print being displayed when multiple are available. I would assume all would be used but feel free to bring it up in the discussion: https://github.com/elifesciences/issues/issues/4702